0008128 - ✨ Pouvoir disposer de plusieurs signature au sein d'une même BAL

CottonDamien opened the pull request.

Commit:
fix(notifications): :bug: Correction sur le nombre de notifications restantes

Commit:
fix(notifications): :bug: Correction-nombre-max-notifications

Commit:
fix

Commit:
feat(signature): :sparkles: Possibilité d'enregistrer plusieurs signatures et de paramétrer leur utilisation

tpayen commented on skins/mel_elastic/templates/compose.html:
```diff
@@ -257,6 +258,31 @@ <h3 id="aria-label-attachmentmenu" class="voice"><roundcube:label name="arialabe
+    <li role="menuitem" id="sigmenu-full">
+      <a href="#"
+         onclick="rcmail.env.signature_type = 'full'; rcmail.command('insert-sig'); return false;">
+        <span class="sig-checkmark" style="display:none">✓ </span>Signature complète
```
Mettre les textes en localisation plutôt

tpayen commented on skins/mel_elastic/templates/compose.html:
```diff
@@ -257,6 +258,31 @@ <h3 id="aria-label-attachmentmenu" class="voice"><roundcube:label name="arialabe
+  <ul class="menu listing" role="menu">
+    <li role="menuitem" id="sigmenu-full">
+      <a href="#"
+         onclick="rcmail.env.signature_type = 'full'; rcmail.command('insert-sig'); return false;">
```
ça pourrait être propose de créer des commandes 'insert-sig-full', 'insert-sig-medium' et 'insert-sig-simple'

Rotomeca replied:
Si on utilise l'attribut onclick, c'est plus lisible de n'avoir qu'une seule action.
Si tu as un `;` dans un onclick, préfère la création de fonction ou de commandes.

Comme dit @tpayen, mieux vaut créer les commandes `insert-sig-full`, `insert-sig-medium` et 
`insert-sig-simple`

Rotomeca commented on skins/mel_elastic/templates/compose.html:
```diff
@@ -257,6 +258,31 @@ <h3 id="aria-label-attachmentmenu" class="voice"><roundcube:label name="arialabe
+    <li role="menuitem" id="sigmenu-intermediaire">
+      <a href="#"
+         onclick="rcmail.env.signature_type = 'intermediaire'; rcmail.command('insert-sig'); return false;">
+        <span class="sig-checkmark" style="display:none">✓ </span>Signature intermédiaire
```
Localisation

Rotomeca commented on skins/mel_elastic/templates/compose.html:
```diff
@@ -257,6 +258,31 @@ <h3 id="aria-label-attachmentmenu" class="voice"><roundcube:label name="arialabe
+    <li role="menuitem" id="sigmenu-simple">
+      <a href="#"
+         onclick="rcmail.env.signature_type = 'simple'; rcmail.command('insert-sig'); return false;">
+        <span class="sig-checkmark" style="display:none">✓ </span>Signature simple
```
Localisation

Rotomeca commented on skins/mel_elastic/templates/compose.html:
```diff
@@ -257,6 +258,31 @@ <h3 id="aria-label-attachmentmenu" class="voice"><roundcube:label name="arialabe
+    </li>
+    <li role="menuitem" id="sigmenu-intermediaire">
+      <a href="#"
+         onclick="rcmail.env.signature_type = 'intermediaire'; rcmail.command('insert-sig'); return false;">
```
`insert-sig-medium`

Rotomeca commented on skins/mel_elastic/templates/compose.html:
```diff
@@ -257,6 +258,31 @@ <h3 id="aria-label-attachmentmenu" class="voice"><roundcube:label name="arialabe
+    </li>
+    <li role="menuitem" id="sigmenu-simple">
+      <a href="#"
+         onclick="rcmail.env.signature_type = 'simple'; rcmail.command('insert-sig'); return false;">
```
`insert-sig-simple`

Rotomeca reviewed (changes requested):
voila

Commit:
fix(mails): :bug: Mauvais mail lors d'une auto invitation à un évènement

Lorsqu'un organisateur crée un evt qu'il soit directement dans son agenda ou lié à un EDT et que son agenda est partagé, les non-participants peuvent s'inviter  L'organisateur a bien reçu un mail de notification lui indiquant que [xxxx a rejoint votre évènement xxxx!]  Une action est attendue de la part de l'organisateur, mais il n'y a aucun bouton qui permet d'accepter ou pas  Du moment où l'agenda est partagé n'importe qu'elle personne ayant le partage peu assister à la réunion

tpayen committed:
feat(mel): support non ldap standard driver

tpayen committed:
fix(mel): standard driver error

tpayen committed:
fix(metapage): check if flags exist in message

Fix #395

tpayen committed:
fix(mel): wrong if

Commit:
refactor(signature): :recycle: Modification du code comme demandé lors de la précédente review + possibilité de choisir pas de signature

CottonDamien commented:
Suite bug rebase cloture PR et création d'une nouvelle
